webservices/tasks/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 # Feature and dev are sharing the same RDS box so we only want dev to update
 schedule = {}
-if not os.getenv('FEATURE'):
+if settings.get('space_name') != 'FEATURE':
```

Comment:
Quick catch, the value is lower case in Cloud Foundry - it might be worth explicitly lower-casing the value pulled back just to be sure, e.g., `settings.get('space_name').lower() != 'feature'`, just in case that ever changes on us without us knowing.